pysquared/hardware/radio/manager/rfm9x.py

```diff
@@ -122,7 +122,6 @@ def modify_config(self, radio_config: RadioConfig) -> None:
 
             if self._radio.spreading_factor > 9:
                 self._radio.preamble_length = self._radio.spreading_factor
-                self._radio.low_datarate_optimize = 1
```

Comment:
Removed because `low_datarate_optimize` is set by the rfm lib when setting spreading factor: https://github.com/adafruit/Adafruit_CircuitPython_RFM/blob/91b9767ff56f537de736ca293abf1771c7ae2cea/adafruit_rfm/rfm9x.py#L481-L485